docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
+
+The ``landice/ismip6_forcing`` test group processes (i.e., remaps and renames)
+the atmospheric and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The processed data is used to force MALI in future ice sheet
+simulations. The test group includes three test cases, ``atmosphere``,``ocean_basal`` and
+``ocean_thermal``. Each test case contains one main step, called
+``ProcessSMB``, ``ProcessBasalMelt`` and ``ProcessThermalForcing``.
+Each step has the local methods (functions) of remapping and
+renaming the original ismip6 data to the format that MALI can incorporate in
+its forward simulations. In remapping the data, all test cases import the
+method ``build_mapping_file`` to create or use scrip files
+of the source (ISMIP6) and destination (MALI) mesh depending on the existence
+of a mapping file. Approximated time for processing a single forcing file
+on Cori (single core) is 2, 7, and 1 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
+Once the users get access to the data on Globus (https:www.globus.org), they
+need to create a base directory that is equivalent to the GHub globus endpoints
+(e.g., ``GHub-ISMIP6-Forcing`` for the 2100CE projection protocol and
+``ISMIP6-Projections-Forcing-2300`` for the 2300CE projection protocol.)
+Users also need to manually download the files in the same directory name and
+structure as on the endpoint (including the ``imbie2_basin_numbers_8km.nc``
+file in the ``Ocean_Forcing/imbie2/`` directory, which is provided in the
+globus ``GHub-ISMIP6-Forcing`` endpoint.
+
+In addition to the ISMIP6 forcing files and the basin numbers file, users have
+to download the RACMO modern climatology data `Jan Melchior van Wessem et al
+. (2018) https://tc.copernicus.org/articles/12/1479/2018/#section7>`_.
```

Comment:
```suggestion
to download the RACMO modern climatology data 
`Jan Melchior van Wessem et al. (2018) https://tc.copernicus.org/articles/12/1479/2018/#section7>`_.
```
A link like this can't be divided across lines.

Reply:
I looked at the paper but it wasn't clear to me which data I needed to download.  I think this needs to be expanded to more thoroughly explain which data set is needed (which specific files) and how to get it from this link.

Reply:
The closest I found was this but it's only until 2016:
https://www.projects.science.uu.nl/iceclimate/publications/data/2018/vwessem2018_tc/RACMO_Yearly/

Reply:
Perhaps this is using a dataset that Michiel van den Broeke sent me by email a few years again, which I put up on Cori. The 2018 TC paper is still the correct reference.

Reply:
We could also use data from here: https://zenodo.org/record/6602723#.YyERTcHMIbY
This "constitutes an update of the simulation forced from 1979-2018 by ERA-Interim reported in van Wessem et al., 2018", but I don't know if 1979–2018 are different here or not. Would be worth taking a look in order to cite a publicly available dataset rather than one sent by email.

Reply:
For now, I've put a place holder in the document saying users should reach out to Trevor to acquire the RACMO data file.

Reply:
This has been resolved as we now have the file on our database ` https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-albany-landice/`